client/packages/common/src/ui/layout/tables/material-react-table/useNonPaginatedMaterialTable.ts

```diff
@@ -36,6 +42,12 @@ export const useNonPaginatedMaterialTable = <T extends MRT_RowData>({
         verticalAlign: 'bottom',
       },
     },
+    muiTableContainerProps: {
+      sx: {
+        maxHeight: '100%',
+        overflow: 'auto',
+      },
+    },
```

Comment:
This keeps the *content* part of the tablel scrollable, but leaves the headers fixed